packages/angular_devkit/build_angular/src/angular-cli-files/models/webpack-configs/worker.ts

```diff
@@ -0,0 +1,29 @@
+/**
+ * @license
+ * Copyright Google Inc. All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+import { virtualFs } from '@angular-devkit/core';
+import { Stats } from 'fs';
+import { resolve } from 'path';
+import { Configuration } from 'webpack';
+import { WebpackConfigOptions } from '../build-options';
+import { getTypescriptWorkerPlugin } from './typescript';
+
+const WorkerPlugin = require('worker-plugin');
+
+
+export function getWorkerConfig(wco: WebpackConfigOptions): Configuration {
+  const { buildOptions } = wco;
+  const workerTsConfigPath = buildOptions.workerTsConfig
+    ? resolve(wco.root, buildOptions.workerTsConfig)
+    : undefined;
+
+  return {
+    plugins: [new WorkerPlugin({
```

Comment:
The child compilation where this plugin runs doesn't seem to run the environment hook.

This messes up virtual file system creation for AngularCompilerPlugin and breaks compilation.

Reply:
Fixed by making the webpack plugin in child compilations pick up the parent WebpackCompilerHost from the VirtualFileSystemDecorator.